src/ci/docker/host-x86_64/x86_64-gnu-tools/Dockerfile

```diff
@@ -15,6 +15,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   sudo \
   xz-utils \
   tidy \
+  zlib1g-dev \
```

Comment:
since the lld zlib change was reverted, I assume this is not needed anymore in the two dockerfiles?

Reply:
Good catch. We should remove it from here, but I suppose that we can keep it in the second one, given that we want to test zlib there (although the current test doesn't really test anything, so it would have to be done in a follow-up PR anyway :man_shrugging:).

Reply:
zlib should already be working according to the previous comments, so we'd just check that it's indeed the case, and that the existing test case (not the one added in this PR) is executed, works as expected, etc.

Reply:
I think that zlib was indeed working because we have zlib in our dist x64 Linux runner, but since it wasn't installed in the test runner, if a test actually checks that zlib is present, that test should fail, I think.

Reply:
Yeah, maybe. Maybe it's executed on the dist builder, in the post-dist artifacts tests 🤷